config/initializers/teaspoon.rb

```diff
@@ -0,0 +1,5 @@
+unless Rails.env.production?
+  class Teaspoon::SuiteController < ActionController::Base
+    skip_before_action :handle_two_factor_authentication
```

Comment:
I don't think so, we need to be able to override the 2fa check in the browser and the command line.